services/server/src/server/services/storageServices/AbstractDatabaseService.ts

```diff
@@ -155,6 +189,12 @@ export default abstract class AbstractDatabaseService {
     recompiledContract: CheckedContract,
     match: Match,
   ): Promise<Database.DatabaseColumns> {
+    const {
+      sha256OnchainCreationBytecode,
+      sha256OnchainRuntimeBytecode,
+      sha256RecompiledCreationBytecode,
+      sha256RecompiledRuntimeBytecode,
+    } = this.getSha256Bytecodes(recompiledContract, match);
     const {
```

Comment:
Shouldn't we be getting the sha256 value directly from the DB response instead of calculating ourselves? See the comment in `database-utils.ts`